haeun/RestApiStudy/src/main/java/com/example/restapistudy/controller/BoardController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.restapistudy.controller;
+
+
+import com.example.restapistudy.dto.BoardDto;
+import com.example.restapistudy.service.BoardService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class BoardController {
+    private final BoardService boardService;
+
+    // 생성자 주입
+    public BoardController(BoardService boardService) {
+        this.boardService = boardService;
+    }
+
+    // 게시글 저장
+    @PostMapping("board")
+    public void save(@RequestBody BoardDto boardDto){
+        boardService.saveBoard(boardDto);
+    }
+
+    // 특정 게시글 조회
+    @GetMapping("board/{id}")
+    public BoardDto getBoard(@PathVariable Long id){
+        return boardService.findBoardById(id);
+    }
+    // 전체 게시글 조회
+    @GetMapping("board/list")
+    public List<BoardDto> getBoardList(){
```

Comment:
```suggestion
    @GetMapping("board/{id}")
    public BoardDto getBoard(@PathVariable Long id){
        return boardService.findBoardById(id);
    }
    
    // 전체 게시글 조회
    @GetMapping("board/list")
    public List<BoardDto> getBoardList(){
```

제가 말씀드린 개행은 메서드와 메서드 사이 개행!